optashift-employee-rostering-shared/src/main/resources/META-INF/persistence.xml

```diff
@@ -3,15 +3,17 @@
              version="2.1" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
   <persistence-unit name="optashift-employee-rostering-persistence-unit" transaction-type="JTA">
     <description>Hibernate Persistence Unit</description>
-    <jta-data-source>java:jboss/datasources/ExampleDS</jta-data-source>
+    <jta-data-source>${optashift.rostering.persistence.datasource:java:jboss/datasources/ExampleDS}</jta-data-source>
     <exclude-unlisted-classes>false</exclude-unlisted-classes>
     <properties>
       <property name="hibernate.hbm2ddl.auto" value="create"/>
-      <!-- <property name="hibernate.show_sql" value="true"/> -->
+      <property name="hibernate.show_sql" value="false"/> 
       <property name="hibernate.format_sql" value="true"/>
       <property name="hibernate.transaction.flush_before_completion" value="true"/>
-      <property name="hibernate.dialect" value="org.hibernate.dialect.H2Dialect"/>
+      <property name="hibernate.dialect" value="${optashift.rostering.persistence.dialect:org.hibernate.dialect.H2Dialect}"/>
```

Comment:
See above (change to ${optashift.rostering.persistence.dialect} and set the default value in properties